actions/pack_action.go

```diff
@@ -77,16 +78,14 @@ func (pf *PackAction) Run(context *debos.DebosContext) error {
 	outfile := path.Join(context.Artifactdir, pf.File)
 
 	command := []string{"tar"}
-	if usePigz == true {
-		command = append(command, "cf")
-	} else {
-		command = append(command, "cf" + tarOpts[pf.Compression])
-	}
+	command = append(command, "cf")
 	command = append(command, outfile)
 	command = append(command, "--xattrs")
 	command = append(command, "--xattrs-include=*.*")
 	if usePigz == true {
 		command = append(command, "--use-compress-program=pigz")
+	} else if pf.Compression != "none" {
```

Comment:
rather then checking for None; maybe check if `tarOpts[pf.Compression] != ""` as that's what you're really after